### ISSUE TITLE
Cap use for non-repeating modules is wrong.

### ISSUE THREAD

**Issue by Neugeniko** [pyfa-org/Pyfa#1482]
### Expected behavior:
The cap use for MJD and Assault DC shows average cap/sec use over the total of cycle time and reactivation delay.

### Actual behavior:
The cap/sec is shown only for the cycle time resulting in a larger cap drain than average.

### Detailed steps to reproduce:
Use the fit below and note the cap drain is -2.46 GJ per sec (32 GJ/13 sec cycle time)
It should be 32 GJ/(13 sec cycle time + 150sec reactivation delay)= 0.20 GJ per sec

### Fits involved in EFT format (Edit > To Clipboard > EFT):
[Retribution, Retribution fit]

Assault Damage Control I
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]

[Empty Med slot]
[Empty Med slot]

[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]

[Empty Rig slot]
[Empty Rig slot]

### Release or development git branch? Please note the release version or commit hash:
py3 or master

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
win 10

**Comment by Neugeniko** [comment]
There is a incomplete implementation for calculating the average cycle time in module.py. It forgets to modify the (average) duration of the module if it has numshots = 0 like the MJD and Assault DC.

```
    def cycleTime(self):
        # Determine if we'll take into account reload time or not
        factorReload = self.owner.factorReload if self.forceReload is None else self.forceReload

        numShots = self.numShots
        speed = self.rawCycleTime

        if factorReload and self.charge:
            raw_reload_time = self.reloadTime
        else:
            raw_reload_time = 0.0

        # Module can only fire one shot at a time, think bomb launchers or defender launchers
        if self.disallowRepeatingAction:
            if numShots > 1:
                """
                The actual mechanics behind this is complex.  Behavior will be (for 3 ammo):
                    fire, reactivation delay, fire, reactivation delay, fire, max(reactivation delay, reload)
                so your effective reload time depends on where you are at in the cycle.

                We can't do that, so instead we'll average it out.

                Currently would apply to bomb launchers and defender missiles
                """
                effective_reload_time = ((self.reactivationDelay * (numShots - 1)) + max(raw_reload_time, self.reactivationDelay, 0)) / numShots
            else:
                """
                Applies to MJD/MJFG
                """
                effective_reload_time = max(raw_reload_time, self.reactivationDelay, 0)
                ****HERE***
        else:
            """
            Currently no other modules would have a reactivation delay, so for sanities sake don't try and account for it.
            Okay, technically cloaks do, but they also have 0 cycle time and cap usage so why do you care?
            """
            effective_reload_time = raw_reload_time

        if numShots > 0 and self.charge:
            speed = (speed * numShots + effective_reload_time) / numShots

        return speed
```
So just at - speed = speed + effective_reload_time - where marked **HERE**.

**Comment by Neugeniko** [comment]
I'll put in a PR.

**Comment by Neugeniko** [comment]
Fix is dependant on #1480.
